cage.c

```diff
@@ -232,6 +234,7 @@ usage(FILE *file, const char *cage)
 		" -m extend Extend the display across all connected outputs (default)\n"
 		" -m last Use only the last connected output\n"
 		" -s\t Allow VT switching\n"
+		" -n\t Enable numlock\n"
```

Comment:
One more CLI arg... configuration file support should really be added to Cage (first?).